frontend/src/pages/chat/Chat.tsx

```diff
@@ -63,6 +63,7 @@ const Chat = () => {
   const [clearingChat, setClearingChat] = useState<boolean>(false)
   const [hideErrorDialog, { toggle: toggleErrorDialog }] = useBoolean(true)
   const [errorMsg, setErrorMsg] = useState<ErrorMessage | null>()
+  const [logo, setLogo] = useState('')
```

Comment:
Just for my understanding, can you explain how this works?  Mainly I don't understand how `useState` with empty string as input works, and also I don't see a reference to `setLogo` anywhere.  It's possible I have missed something.